File: app/src/main/java/kr/zagros/shwan/moviemvvm/DataSource/Client/ApiService.java
package kr.zagros.shwan.moviemvvm.DataSource.Client;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiService {

    /**
     * Get a list of movies in theatres.
     */

    @GET("v1/movies")
    Call<ResponseBody> getMovies(@Query("page") long page);

}

File: app/src/main/java/kr/zagros/shwan/moviemvvm/DataSource/Client/ApiService.kt
package kr.zagros.shwan.moviemvvm.DataSource.Client

import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiService {

    @get:GET(".")
    val a: Call<ResponseBody>

    /**
     * Get a list of movies in theatres.
     */

    @GET("v1/movies")
    fun getMovies(@Query("page") page: Long): Call<ResponseBody>

}
